apps/ewallet_config/config/config.exs

```diff
@@ -12,12 +12,19 @@ config :ewallet_config,
   },
   default_settings: %{
     # Global Settings
-    "base_url" => %{key: "base_url", value: "", type: "string", position: 100},
+    "base_url" => %{
+      key: "base_url",
+      value: "",
+      type: "string",
+      position: 100,
+      description: "The domain name where this eWallet is accessible."
+    },
     "redirect_url_prefixes" => %{
       key: "redirect_url_prefixes",
       value: [],
       type: "array",
-      position: 101
+      position: 101,
+      description: "The whitelisted domain names the eWallet can redirect to (for password resets, etc.)."
```

Comment:
Can we rephrase to avoid the word "domain name"? Because the settings actually need a URL. E.g.

> The URL prefixes where the eWallet are allowed to redirect to (for password resets, etc.)